app/javascript/utils/Patient.js

```diff
@@ -61,4 +67,36 @@ function formatRace(patient) {
   return raceArray.length === 0 ? '--' : raceArray.join(', ');
 }
 
-export { formatName, formatNameAlt, formatPhoneNumber, formatRace };
+function isMinor(date) {
+  return moment(date, 'YYYY-MM-DD').isAfter(moment().subtract(18, 'years')); 
+}
+
+/**
+ * Formats values in the date of birth column to be human readable and include whether that DOB indicates a minor.
+ * @param {Object} data - Data about the cell this filter is called on.
+ */
+function formatDateOfBirth(data) {
+  const rowData = data.rowData;
+  if (!!rowData?.dob && isMinor(rowData.dob)) {
+    return (
+      <div>
+        <IconMinor patientId={rowData.id.toString()} customClass={'float-right ml-1'} />
+        {formatDate(rowData.dob)}
+      </div>
+    );
+  }
+  if (!!rowData?.date_of_birth && isMinor(rowData.date_of_birth)) {
+    return (
+      <div>
+        <IconMinor patientId={rowData.id.toString()} customClass={'float-right ml-1'} />
+        {formatDate(rowData.date_of_birth)}
+      </div>
+    );
+  }
```

Comment:
It's a bit awkward to have this method coupled to the `CustomTable` `rowData` structure, and to have to check for both the `dob` and `date_of_birth` keys.

Can you instead simplify this method to only take the primitive parameters it needs (I `id` and `date_of_birth` strings), and then put separate `filter` methods in each of the components that use this (`PatientsTable`, `HouseholdMemberTable`) which just act as shims for this utility method?

Reply:
I definitely would prefer this to just take in a dob. let each table worry about passing in the correct field and not have this rely on rowData at all